pom.xml

```diff
@@ -385,8 +385,8 @@
         <osgi.util.tracker.imp.pkg.version.range>[1.5.1, 2.0.0)</osgi.util.tracker.imp.pkg.version.range>
 
         <!-- Carbon Kernel version -->
-        <carbon.kernel.version>4.9.17</carbon.kernel.version>
-        <carbon.kernel.feature.version>4.9.0</carbon.kernel.feature.version>
+        <carbon.kernel.version>4.10.24</carbon.kernel.version>
```

Comment:
Need to update this with the new release version with the changes in

Reply:
Updated to the latest kernel release 4.10.26[1]

[1] https://github.com/wso2/carbon-kernel/tree/v4.10.26